lib/services/local/events.go

```diff
@@ -214,6 +215,10 @@ func (e *EventsService) NewWatcher(ctx context.Context, watch types.Watch) (type
 			parser = newBotInstanceParser()
 		case types.KindDevice:
 			parser = newDeviceParser()
+		case types.KindAccessGraphSecretPrivateKey:
```

Comment:
Since we do not store private keys here are mentioned here https://github.com/gravitational/teleport/pull/43543#discussion_r1670845974 I'd rename it to better reflect the purpose.